posthog/models/feedback/survey.py

```diff
@@ -62,3 +52,16 @@ class Meta:
     end_date: models.DateTimeField = models.DateTimeField(null=True)
     updated_at: models.DateTimeField = models.DateTimeField(auto_now=True)
     archived: models.BooleanField = models.BooleanField(default=False)
+
+
+@mutable_receiver(post_save, sender=Survey)
```

Comment:
@neilkakkar pinging you for a look now before your day ends and you're also off tomorrow 😄

Reply:
We also want to handle the case when someone deletes a running survey & say it's the last running survey

```suggestion
@mutable_receiver([post_save, post_delete], sender=Survey)
```